conda_smithy/configure_feedstock.py

```diff
@@ -1630,6 +1634,27 @@ def get_migrations_in_dir(migrations_root):
     return res
 
 
+def get_cfp_migration():
+    """Load the migrations from the central migration store.
+
+    Returns
+    -------
+    migrations : dict
+        The migrations from the central store loaded
+    """
+    # TODO: replace with a more flexable key value store? (dynamoDB?)
+    with tempfile.TemporaryDirectory() as td:
+        tdn = td.name
+        # TODO: potentially pull git args from a config (in case you wanted
+        #  to point at a different remote/branch/etc.)
+        subprocess.call(['git', 'clone', '--depth', '1', 'https://github.com/conda-forge/conda-forge-pinning-feedstock.git', tdn])
+        cfp_migration_dir = os.path.join(tdn, 'recipe', 'migrations')
+        if not os.path.exists(cfp_migration_dir):
+            cfp_migration_dir = None
+        migrations = get_migrations_in_dir(cfp_migration_dir)
+    return migrations
```

Comment:
You are returning the file paths from a temporary directory which is going to get deleted after the context manager exists

Reply:
Ah, sorry I thought `get_migrations_in_dir` returned the actual migrations

Reply:
Now the paths should be valid until they are parsed at which time we no longer need them.